### ISSUE TITLE
I have no idea what I'm doing

### ISSUE THREAD

**Issue by iamleosamson** [florianfesti/midi2ariston#1]
Hi!

I started restoring an Ariston organette today, and I stumbled across this project while I was looking up some specifications, and it looks like more than I could have ever hoped for.

Unfortunately, I'm an idiot. I don't know the first thing about how I could deploy these scripts to make music happen, and every tutorial I look up just leaves me more confused.

If you've got it up and running and it's no trouble, is there any chance I could send you some midi files and you could send me the image it outputs?

**Comment by florianfesti** [comment]
Yes, sure. Be aware that you typically need to re-arrange the midi to fit the tunes playable on the organette. Which should be
```
  45 A_3
  47 B_3
  50 D_4
  52 E_4
  57 A_4
  59 B_4
  61 Cs_5
  62 D_5
  64 E_5
  66 Fs_5
  68 Gs_5
  69 A_5
  71 B_5
  73 Cs_6
  74 D_6
  75 Ds_6
  76 E_6
  78 Fs_6
  79 G_6
  80 Gs_6
  81 A_6
  83 B_6
  85 Cs_7
  86 D_7
```
